IPs/RSKIP427.md

```diff
@@ -0,0 +1,45 @@
+---
+rskip: 427
+title: Express amount value as weis in peg-out related events
+created: 17-APR-24
+author: MI
+purpose: Usa
+layer: Core
+complexity: 1
+status: Draft
+description: 
+---
+
+|RSKIP          |427           |
+| :------------ |:-------------|
+|**Title**      |Express amount value as weis in peg-out related events |
+|**Created**    |17-APR-24 |
+|**Author**     |MI |
+|**Purpose**    |Usa |
+|**Layer**      |Core |
+|**Complexity** |1 |
+|**Status**     |Draft |
+
+## Abstract
+
+This RSKIP proposes a change in the unit used in `amount` field of `release_request_received` and `release_request_rejected` Bridge events. The proposal is to express the amount in **weis** instead of **satoshis**.
+
+## Motivation
+
+In the original specification [1] it was specified that `release_request_received` and `release_request_rejected` Bridge events should express the _amount_ field in weis.
+
+## Specification
+
+When `release_request_received` or `release_request_rejected` Bridge events are emitted, the _amount_ field should include the value in weis that the user sent to the Bridge.
+
+## Backward Compatibility
+
+This change is a hard fork and therefore all full nodes must be updated.
```

Comment:
We may include `Since the same amount is different if expressed in weis rather than satoshis, this change is a hard fork...`
Just a thought

Reply:
Mmm not sure, I think that should be already understood by the specification